docs/maya/ribbon.md

```diff
@@ -56,6 +56,13 @@ Follow this steps to create an AndRibbonMuscle deformer:
 
 ### Advanced Settings
 
+#### Initialization Settings
```

Comment:
Just a quick reminder that we will have to update the screenshots of the simple setup as it is also required for the Debug Fibers AET entry. maybe we can document this and leave it for the very end so we don't have to do work twice